--- build/spike.nopurge.pt	2026-07-29 11:38:07.292628065 +0200
+++ build/spike.pt	2026-07-29 11:38:07.290485845 +0200
@@ -1,7 +1,5 @@
 <!DOCTYPE html>
-
 <html lang="${lang}" dir="ltr" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
-  
   <head>
     <meta charset="utf-8">
     <meta name="x-apple-disable-message-reformatting">
@@ -12,29 +10,13 @@
     <!--[if mso]>
       <style>
         td,th,div,p,a,h1,h2,h3,h4,h5,h6 {font-family: "Segoe UI", sans-serif; mso-line-height-rule: exactly;}
-        .mso-break-all {word-break: break-all;}
       </style>
     <![endif]-->
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&display=swap" rel="stylesheet" media="screen">
-    <style>
-      /*! tailwindcss v4.3.3 | MIT License | https://tailwindcss.com */
-      .absolute {
-        position: absolute !important;
-      }
-      .block {
-        display: block !important;
-      }
-      .inline {
-        display: inline !important;
-      }
-      .table {
-        display: table !important;
-      }
-    </style>
   </head>
-  <body xml:lang="${lang}" class="m-0 p-0 size-full word-break-break-word bg-gray-50" style="margin: 0; width: 100%; height: 100%; background-color: #f9fafb; padding: 0; word-break: break-word;">
+  <body xml:lang="${lang}" style="margin: 0; width: 100%; height: 100%; background-color: #f9fafb; padding: 0; word-break: break-word;">
     <div style="display:none;">
       ${preheader}&#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847;
       &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847; &#8199;&#65279;&#847;
@@ -63,24 +45,9 @@
         </xml>
       <![endif]-->
     </span>
-    <div
-      role="article"
-      aria-roledescription="email"
-      lang="${lang}"
-      dir="ltr"
-      style="font-size: medium; font-family: Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Noto Sans', Ubuntu, Cantarell, 'Helvetica Neue', sans-serif; font-size: max(16px, 1rem);"
-      class="font-size-max16px_1rem font-inter"
-    >
-      
-      
-      <style>
-        .spike-purge-victim {
-          color: #059669;
-          font-weight: 700;
-        }
-      </style>
+    <div role="article" aria-roledescription="email" lang="${lang}" dir="ltr" style="font-size: medium; font-family: Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Noto Sans', Ubuntu, Cantarell, 'Helvetica Neue', sans-serif; font-size: max(16px, 1rem);">
       <!--[if mso]><table role="none" cellpadding="0" cellspacing="0" style="width: 576px" align="center"><tr><td style="padding: 24px"><![endif]-->
-      <div class="m-0 mx-auto max-w-xl p-6" style="margin: 0 auto; max-width: 576px; padding: 24px;">
+      <div class="mx-auto max-w-xl p-6" style="margin: 0 auto; max-width: 576px; padding: 24px;">
         
         <div i18n:domain="imio.emailkit">
           
